test/gtest/ucp/test_ucp_am.cc

```diff
@@ -702,6 +704,68 @@ UCS_TEST_P(test_ucp_am_nbx_eager_memtype, basic)
 UCP_INSTANTIATE_TEST_CASE_GPU_AWARE(test_ucp_am_nbx_eager_memtype)
 
 
+class test_ucp_am_nbx_eager_data_release : public test_ucp_am_nbx {
+public:
+    test_ucp_am_nbx_eager_data_release()
+    {
+        modify_config("RNDV_THRESH", "inf");
+        modify_config("ZCOPY_THRESH", "inf");
```

Comment:
maybe test different combinations?

Reply:
i have separate rndv class, which tests data release.
i'd not combine them, because data release semantic is different for eager and rendezvous

Reply:
so this class is only for eager?
why need to disable zcopy then?
and why need to set RNDV_THRESH, given that we can control eager/rndv mode by AM API?

Reply:
disable zcopy, because of `fragment_size()` func, which calculates fragments size
flag for controlling AM API is tested separately, this is just another way to disable rndv

Reply:
can we update it to test zcopy as well?
should aim to have more coverage

Reply:
we can, but sending eager protocol does not affect receive flow at all (by design).
Here we test single and multi-fragment cases - seems this is enough, wdyt?